sdk/core/azure-core/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+- Fixed issue InteractiveBrowserCredential does not hand over to next credential in chain if no browser is supported.
```

Comment:
`Fixed an issue where InteractiveBrowserCredential failed to switch to the next credential in the chain when no browser was supported.`

Reply:
Please consider adding a [#bugNumber] too